tests/test_main.cpp

```diff
@@ -170,6 +170,34 @@ int test_main(int argc, char *argv[]) {
 		}
 	}
 
+#if defined(__has_feature)
+#if __has_feature(thread_sanitizer)
+	{
+		Vector<String> exclude_list;
+
+		// 1 Padding comment to reduce merge conflicts
+		//exclude_list.push_back("*Resource*");
```

Comment:
Did you mean to leave this comment in? `*Resource*` seems pretty broad.

Should we use [TSAN Exclusions](https://github.com/google/sanitizers/wiki/ThreadSanitizerFlags#compiler-flags-llvm-specific) instead?

If these are known races, it might be nice to link to open issues if they exist.

Reply:
> Did you mean to leave this comment in? `*Resource*` seems pretty broad.
> 

Some of these were fixed in the meantime while I was working on this, but I left them in (albeit commented out) just in case there's a resurgence.

This probably isn't necessary anymore since the TSAN exclusions format makes it very easy to track down and record where errors are happening.